example/09_convnd_fwd/convnd_fwd_xdl_fp16.cpp

```diff
@@ -320,17 +371,17 @@ int main(int argc, char* argv[])
         {
         case 3: {
             auto ref_conv = ReferenceConvNDFwdInstance<3>();
-            verify_f(ref_conv);
+            return verify_f(ref_conv);
             break;
         }
         case 2: {
             auto ref_conv = ReferenceConvNDFwdInstance<2>();
-            verify_f(ref_conv);
+            return verify_f(ref_conv);
             break;
```

Comment:
it will cause ```reachable-code-break``` warning. So the ```break``` might be deleted.

Reply:
fixed